azure_functions_worker/mmap_handler/file_reader.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import mmap
+import os
+import struct
+from typing import Optional
+from .memorymappedfile_constants import MemoryMappedFileConstants as consts
+from .file_accessor_factory import FileAccessorFactory
+
+
+class FileReader:
```

Comment:
Given that on Windows the mmap is probably not a file, I'm not sure File is the best naming.

Reply:
Agreed - will change to `SharedMemoryMap`.